src/test/unit/engine/RESTapiTest.cpp

```diff
@@ -225,4 +225,55 @@ TEST(RESTapiTest, test_delete) {
   threadObj.join();          // wait until server thread has stopped.
 }
 
+
+TEST(RESTapiTest, alternative_ids) {
+  std::thread threadObj(serverThread); // start REST server
+  std::this_thread::sleep_for(std::chrono::seconds(1)); // give server time to start
```

Comment:
> give server time to start 

could the server return something like `STATUS: OK` once it's up and running?

Reply:
> could the server return something like STATUS: OK

This is actually a web server and web servers cannot originate a message. 
Giving it one second is probably much more than we need but we need something so the process will yield to the server thread for initialization before allowing the creating thread to continue.  The originating thread will open the network connection before posting the first request so the timing is already handled.

I will change the delay to 1ms.